$MirOS: ports/www/firesomething/patches/patch-security_nss_lib_softoken_pkcs11_c,v 1.1.2.1 2008/09/06 19:29:01 bsiegert Exp $
--- security/nss/lib/softoken/pkcs11.c.orig	Fri Jul  4 02:02:03 2008
+++ security/nss/lib/softoken/pkcs11.c	Tue Dec  2 19:03:04 2008
@@ -2549,7 +2549,7 @@ loser:
         sftk_InitFreeLists();
     }
 
-#if defined(XP_UNIX) && !defined(NO_PTHREADS)
+#if defined(XP_UNIX) && !defined(NO_PTHREADS) && !defined(__MirBSD__)
     if (CKR_OK == crv) {
         pthread_atfork(NULL, NULL, ForkedChild);
     }
